dd-sdk-android/src/main/kotlin/com/datadog/android/plugin/DatadogPluginConfig.kt

```diff
@@ -41,5 +41,5 @@ sealed class DatadogPluginConfig(
         DatadogPluginConfig(context, envName, serviceName, TracingFileStrategy.TRACES_FOLDER)
 
     internal class RumPluginConfig(context: Context, envName: String, serviceName: String) :
```

Comment:
Shouldn't you also change the `LogFileStrategy.LOGS_FOLDER` to `LogFileStrategy. AUTHORIZED_FOLDER` and `TracingFileStrategy.TRACES_FOLDER` to `TracingFileStrategy.AUTHORIZED_FOLDER` in this file too ?